libs/blocks/modal/modal.js

```diff
@@ -134,6 +135,7 @@ export async function getModal(details, custom) {
   const dialog = createTag('div', { class: 'dialog-modal', id, role: 'dialog', 'aria-modal': true });
   const loadedEvent = new Event('milo:modal:loaded');
 
+  if (custom && !custom?.title) custom.title = findDetails(window.location.hash, null)?.title;
```

Comment:
If custom?.title is available, custom is Too. You already check for `custom?.`
```suggestion
  if (!custom?.title) custom.title = findDetails(window.location.hash, null)?.title;
```

Reply:
But we have an ! in front of that, won't it enter the if clause then if custom is null and cause an error because we are assigning title to null?

Reply:
Oh your right, disregard the comment!